app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java

```diff
@@ -92,17 +93,18 @@ protected void onCreate(Bundle savedInstanceState) {
      * @param v v is a View, allows usage of on click in xml
      */
     public void editPhoneNumber(View v) {
-        ImageButton cancelButton = (ImageButton) findViewById(R.id.CancelEditPhoneButton);
-        ImageButton saveButton = (ImageButton) findViewById(R.id.PhoneSaveEditButton);
-        ImageButton editButton = (ImageButton) findViewById(R.id.PhoneEditIconImageButton);
+        ImageButton cancelButton = (ImageButton) findViewById(R.id.Button_cancelPhoneEdit);
```

Comment:
This is maybe something to discuss as a group...I've been doing **button_cancelPhoneEdit** instead of **Button_cancelPhoneEdit**. We should maybe decide which convention we want to follow.